amgut/lib/data_access/ag_data_access.py

```diff
@@ -323,7 +340,10 @@ def deleteAGParticipantSurvey(self, ag_login_id, participant_name):
                 self.deleteSample(info['barcode'], ag_login_id)
 
             # Delete last due to foreign keys
-            sql = "DELETE FROM ag_kit_barcodes WHERE survey_id IN %s"
+            sql = "DELETE FROM ag.ag_kit_barcodes WHERE barcode IN %s"
```

Comment:
Seems like the foreign key does not restrict deletion. Could you please check if I created those FK correctly in the patch?